File: ProyectoBaseWebAppSauluwu/app/TipoBulto.php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;

class TipoBulto extends Model
{
    protected $table = 'tipos_bulto';
    protected $fillable = [
        'empresa_id', 'codigo', 'ancho', 'largo_id', 'espesor_id', 'estado', 'creador_id', 'modificador_id'
    ];

    public function largo()
    {
        return $this->belongsTo('App\Largo');
    }

    public function espesor()
    {
        return $this->belongsTo('App\Espesor');
    }

    public function scopeActive($query)
    {
        return $query->where('estado', 'A');
    }
    public function creador()
    {
        return $this->belongsTo('App\User');
    }
    public function modificador()
    {
        return $this->belongsTo('App\User');
    }
    public function scopeCurrent($query, $empresaId)
    {
        return $query->where('empresa_id', $empresaId);
    }
}

File: CosechaApp-main/app/src/main/java/com/techtraining/cosechasapp/db/TipoBulto.java
package com.techtraining.cosechasapp.db;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity
public class TipoBulto {
    @PrimaryKey
    public int id;

    @ColumnInfo(name = "codigo")
    public String codigo;

    @ColumnInfo(name = "largo_id")
    public int largoId;

    @ColumnInfo(name = "espesor_id")
    public int espesorId;

    @ColumnInfo(name = "ancho")
    public double ancho;

    @ColumnInfo(name = "empresa_id")
    public int empresaId;

    @ColumnInfo(name = "estado")
    public String estado;

    @Override
    public String toString() {
        return codigo + " (" + this.ancho + ")";
    }
}
